modules/microshift-lvms-config-example-basic.adoc

```diff
@@ -16,42 +16,20 @@ The following `lvmd.yaml` example file shows a basic LVMS configuration:
 ----
 socket-name: <1>
 device-classes: <2>
-  - name: <3>
-    volume-group: <4>
-    spare-gb: <5>
+  - name: "default" <3>
+    volume-group: "VGNAMEHERE" <4>
+    spare-gb: 0 <5>
     default: <6>
-  - name: hdd
-    volume-group: hdd-vg
-    spare-gb: 10
-  - name: striped
-    volume-group: multi-pv-vg
-    spare-gb: 10
-    stripe: <7>
-    stripe-size: <8>
-  - name: raid
-    volume-group: raid-vg
-    lvcreate-options: <9>
-      - --type=raid1
 ----
-<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  `/run/topolvm/lvmd.sock`.
+<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  '/run/lvmd/lvmd.socket'.
 <2> `map[string]DeviceClass`. The `device-class` settings.
```

Comment:
"map[string]DeviceClass" makes sense to a go programmer, but may not to a sysadmin.

```suggestion
<2> A list of maps for the settings for each `device-class`.
```